easybuild/easyblocks/t/tensorflow.py

```diff
@@ -902,9 +903,11 @@ def build_step(self):
         action_env['PYTHONNOUSERSITE'] = '1'
 
         # TF 2 (final) sets this in configure
-        if LooseVersion(self.version) < LooseVersion('2.0'):
-            if self._with_cuda:
-                self.target_opts.append('--config=cuda')
+        if (LooseVersion(self.version) < LooseVersion('2.0')) and self._with_cuda:
+            self.target_opts.append('--config=cuda')
+        # TF 2.18 with CUDA needs to set cuda_wheel to config
+        if (LooseVersion(self.version) >= LooseVersion('2.18')) and self._with_cuda:
```

Comment:
From how I read the error this is required for non-CUDA builds too. Or did you test this or got other information?